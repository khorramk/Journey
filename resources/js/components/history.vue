<template>
    <div>
        <ul v-for="">
            <li>text</li>
        </ul>
    </div>
</template>

<script>
    export default {
        
    }
</script>

<style lang="scss" scoped>

</style>