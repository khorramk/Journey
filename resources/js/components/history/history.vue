<template>
    <div class="bg-wood h-full-viewport">
      <Nav/>
      <main>
          <Avatar/>
          <Search/>
          <div class="posts-container">
             <Posts />
          </div>
      </main>
    </div>
</template>

<script>
import Avatar from '../dashboard/Avatar';
import Search from '../search/Search';
import Posts from '../Posts/Posts';
import Nav from '../Nav';
    export default {
        components: {
            Avatar,
            Search,
            Nav,
            Posts,
        },
        data() {
            return {
                historyOfPosts: Array(9).fill('test')
            }
        },

    }
</script>

<style lang="scss" scoped>

</style>
