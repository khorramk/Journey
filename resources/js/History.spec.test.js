import { mount } from '@vue/test-utils';
import Vue from 'vue';
import axios from 'axios';
import VueRouter from 'vue-router';
import history from './components/history/history';

const vm = new Vue(history);

it('renders the elements', () => {

    vm.$nextTick().then(()=> {

     expect(vm).toMatchSnapshot();
    }).catch().finally()
  })
