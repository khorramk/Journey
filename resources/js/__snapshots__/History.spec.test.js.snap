// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the elements 1`] = `
Vue {
  "$attrs": Object {},
  "$children": Array [],
  "$createElement": [Function],
  "$listeners": Object {},
  "$options": Object {
    "_base": [Function],
    "components": Object {
      "Avatar": Object {
        "data": [Function],
        "methods": Object {
          "insertAvatar": [Function],
          "onSelectFile": [Function],
          "openSelection": [Function],
          "removeSelection": [Function],
        },
        "render": [Function],
        "staticRenderFns": Array [],
      },
      "Comments": Object {
        "data": [Function],
        "mounted": [Function],
        "render": [Function],
        "staticRenderFns": Array [
          [Function],
        ],
      },
      "Nav": Object {
        "data": [Function],
        "methods": Object {
          "logout": [Function],
          "popLogOutBtn": [Function],
        },
        "render": [Function],
        "staticRenderFns": Array [],
      },
      "Posts": Object {
        "components": Object {
          "Avatar": Object {
            "data": [Function],
            "methods": Object {
              "insertAvatar": [Function],
              "onSelectFile": [Function],
              "openSelection": [Function],
              "removeSelection": [Function],
            },
            "render": [Function],
            "staticRenderFns": Array [],
          },
        },
        "data": [Function],
        "methods": Object {
          "flashModal": [Function],
          "submit": [Function],
        },
        "props": Object {
          "likes": Object {
            "default": [Function],
            "type": [Function],
          },
          "post": Object {
            "default": [Function],
            "type": [Function],
          },
        },
        "render": [Function],
        "staticRenderFns": Array [
          [Function],
        ],
      },
      "Search": Object {
        "render": [Function],
        "staticRenderFns": Array [
          [Function],
        ],
      },
    },
    "data": [Function],
    "directives": Object {},
    "filters": Object {},
    "mounted": Array [
      [Function],
    ],
    "render": [Function],
    "staticRenderFns": Array [],
  },
  "$parent": undefined,
  "$refs": Object {},
  "$root": [Circular],
  "$scopedSlots": Object {},
  "$slots": Object {},
  "$vnode": undefined,
  "_c": [Function],
  "_data": Object {
    "historyOfPosts": "",
  },
  "_directInactive": false,
  "_events": Object {},
  "_hasHookEvent": false,
  "_inactive": null,
  "_isBeingDestroyed": false,
  "_isDestroyed": false,
  "_isMounted": false,
  "_isVue": true,
  "_renderProxy": Vue {
    "$attrs": Object {},
    "$children": Array [],
    "$createElement": [Function],
    "$listeners": Object {},
    "$options": Object {
      "_base": [Function],
      "components": Object {
        "Avatar": Object {
          "data": [Function],
          "methods": Object {
            "insertAvatar": [Function],
            "onSelectFile": [Function],
            "openSelection": [Function],
            "removeSelection": [Function],
          },
          "render": [Function],
          "staticRenderFns": Array [],
        },
        "Comments": Object {
          "data": [Function],
          "mounted": [Function],
          "render": [Function],
          "staticRenderFns": Array [
            [Function],
          ],
        },
        "Nav": Object {
          "data": [Function],
          "methods": Object {
            "logout": [Function],
            "popLogOutBtn": [Function],
          },
          "render": [Function],
          "staticRenderFns": Array [],
        },
        "Posts": Object {
          "components": Object {
            "Avatar": Object {
              "data": [Function],
              "methods": Object {
                "insertAvatar": [Function],
                "onSelectFile": [Function],
                "openSelection": [Function],
                "removeSelection": [Function],
              },
              "render": [Function],
              "staticRenderFns": Array [],
            },
          },
          "data": [Function],
          "methods": Object {
            "flashModal": [Function],
            "submit": [Function],
          },
          "props": Object {
            "likes": Object {
              "default": [Function],
              "type": [Function],
            },
            "post": Object {
              "default": [Function],
              "type": [Function],
            },
          },
          "render": [Function],
          "staticRenderFns": Array [
            [Function],
          ],
        },
        "Search": Object {
          "render": [Function],
          "staticRenderFns": Array [
            [Function],
          ],
        },
      },
      "data": [Function],
      "directives": Object {},
      "filters": Object {},
      "mounted": Array [
        [Function],
      ],
      "render": [Function],
      "staticRenderFns": Array [],
    },
    "$parent": undefined,
    "$refs": Object {},
    "$root": [Circular],
    "$scopedSlots": Object {},
    "$slots": Object {},
    "$vnode": undefined,
    "_c": [Function],
    "_data": Object {
      "historyOfPosts": "",
    },
    "_directInactive": false,
    "_events": Object {},
    "_hasHookEvent": false,
    "_inactive": null,
    "_isBeingDestroyed": false,
    "_isDestroyed": false,
    "_isMounted": false,
    "_isVue": true,
    "_renderProxy": [Circular],
    "_self": [Circular],
    "_staticTrees": null,
    "_uid": 0,
    "_vnode": null,
    "_watcher": null,
    "_watchers": Array [],
    "historyOfPosts": "",
  },
  "_self": [Circular],
  "_staticTrees": null,
  "_uid": 0,
  "_vnode": null,
  "_watcher": null,
  "_watchers": Array [],
  "historyOfPosts": "",
}
`;
